backend/gradle/dependency-locks/compileClasspath.lockfile

```diff
@@ -21,10 +21,15 @@ com.fasterxml.jackson.module:jackson-module-parameter-names:2.11.3
 com.fasterxml.woodstox:woodstox-core:6.2.1
 com.fasterxml:classmate:1.5.1
 com.github.stephenc.jcip:jcip-annotations:1.0-1
-com.google.code.findbugs:jsr305:1.3.9
+com.google.code.findbugs:jsr305:3.0.2
 com.google.code.gson:gson:2.8.6
-com.google.http-client:google-http-client-jackson2:1.21.0
-com.google.http-client:google-http-client:1.21.0
+com.google.errorprone:error_prone_annotations:2.3.4
```

Comment:
I'm confused at the changes to our compile and runtime classpath locks, if there are no changes to our `implementation` dependencies.